proxy/proxy_test.go

```diff
@@ -112,23 +79,23 @@ func TestNewReverseProxyHandler(t *testing.T) {
 	}
 }
 
-func testOptions() *Options {
+func testOptions() *config.Options {
 	authenticateService, _ := url.Parse("https://authenticate.corp.beyondperimeter.com")
 	authorizeService, _ := url.Parse("https://authorize.corp.beyondperimeter.com")
 	configBlob := `[{"from":"corp.example.com","to":"example.com"}]` //valid yaml
```

Comment:
My guess is that this is no longer a valid route and is returning a 404 when attempted to be proxied.